roles/create_mce_operator/tasks/main.yml

```diff
@@ -0,0 +1,201 @@
+- name: Check Storage Classes
+  kubernetes.core.k8s_info:
+    kind: StorageClass
+    kubeconfig: "{{kubeconfig_dest_dir}}{{kubeconfig_dest_filename}}"
+    api: storage.k8s.io/v1
+  register: storage_class_list
+
+- name: Fail if Storage Class is not existing
+  fail:
+    msg: Storage Class is not existing
+  when: storage_class_list.resources | map(attribute='metadata') | selectattr('name', 'in', '{{storage_class_name}}' ) | list | length == 0

```

Comment:
Could you not put the name in the k8s_info task?

Reply:
Yes it didnt work. When I put the name in the k8s_info task, if storage_class doesnt exist, it is not failing and continuing